x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.tsx

```diff
@@ -104,7 +104,7 @@ const RuleActionsOverflowComponent = ({
               data-test-subj="rules-details-export-rule"
               onClick={async () => {
                 closePopover();
-                await exportRulesAction([rule.rule_id], noop, dispatchToaster);
+                await exportRulesAction([rule.rule_id], dispatchToaster, noop);
```

Comment:
nit: We could refactor these action functions to take an object so params are explicit, makes it easier to see which are getting the `noop`.